numpy/linalg/lapack_litemodule.c

```diff
@@ -17,6 +17,8 @@ typedef struct { float r, i; } f2c_complex;
 typedef struct { double r, i; } f2c_doublecomplex;
 /* typedef long int (*L_fp)(); */
 
+int FNAME(MAIN_)() { return 0; }
```

Comment:
The message I received from pip was to put int, so I did so. Shall I change
and resubmit my pull request? -- H

On 15 April 2016 at 13:40, Jaime <notifications@github.com> wrote:

> In numpy/linalg/lapack_litemodule.c
> <https://github.com/numpy/numpy/pull/7550#discussion_r59937622>:
>
> > @@ -17,6 +17,8 @@ typedef struct { float r, i; } f2c_complex;
> >  typedef struct { double r, i; } f2c_doublecomplex;
> >  /* typedef long int (*L_fp)(); */
> >
> > +int FNAME(MAIN_)() { return 0; }
>
> I think you need a void inside the parenthesis to silence the compiler
> warnings and make Travis happy.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/7550/files/faeb2171ed3e73be897a67ddf7be0f8ba978bd5b#r59937622>
>



-- 
OpenPGP: http://hasan.d8u.us/gpg.asc
Sent from my mobile device
Envoyé de mon portable